core/tests/ts-integration/tests/system.test.ts

```diff
@@ -73,9 +73,8 @@ describe('System behavior checks', () => {
     });
 
     test('Should accept transactions with small gasPerPubdataByte', async () => {
-        // The number "10" was chosen because we have a different error for lesser `smallGasPerPubdata`.
-        const smallGasPerPubdata = 10;
-        const senderNonce = await alice.getTransactionCount();
+        // The number "55" was chosen because we have a different error for lesser `smallGasPerPubdata`.
+        const smallGasPerPubdata = 55;
```

Comment:
Why have you removed and changed this? We should differentiate the execution of the test depending on which mode is running? If not the original test will be modified and we expect to make it working in rollup mode